lib/services/discourse-types.d.ts

```diff
@@ -0,0 +1,16 @@
+import {
+    ServiceEmitContext,
+} from './service-types';
+
+export interface DiscourseMessageEmitContext extends ServiceEmitContext {
+    raw: string,
```

Comment:
@hedss I am going to choose to assume that refers to the default settings and not to my code, although equally applicable!